packages/java/fusion-endpoint/pom.xml

```diff
@@ -0,0 +1,268 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>dev.hilla</groupId>
+        <artifactId>hilla-project</artifactId>
+        <version>1.2-SNAPSHOT</version>
+        <relativePath>../../../pom.xml</relativePath>
+    </parent>
+    <groupId>com.vaadin</groupId>
```

Comment:
would it make sense to update this groupID to dev.hilla?

Reply:
Probably not, for sure not in this PR. Also that would be a breaking change for those who depend on it.

Reply:
yeah.. asking as we are having this dependency in platform, once this got changed.. we will need to know that too. :-)

Reply:
It must be changed, we cannot deploy to sonatype under two namespaces at the same time.

Reply:
Will rename in a separate PR then. Thanks for pointing this out.